runelite-client/src/main/java/net/runelite/client/plugins/itemidentification/ItemIdentificationPlugin.java

```diff
@@ -61,4 +65,10 @@ protected void shutDown()
 	{
 		overlayManager.remove(overlay);
 	}
-}
\ No newline at end of file
+
+	ItemIdentification findItemIdentification(final int itemID)
```

Comment:
Actually, now that state is not saved here, simply move this method to overlay and inject ItemManager there